velodrome/README.md

```diff
@@ -58,3 +58,50 @@ time.
 
 As an example, the token counter measures the usage of our github-tokens, and
 has a new value every hour. We can push the new value to InfluxDB.
+
+Adding a new project
+====================
+
+Adding a new project is as simple as adding it to [config.yaml](). Typically,
```

Comment:
Not sure if it's intended, but this link doesn't take you to config.yaml, but to the folder containing it.